scripts/pi-hole/php/header.php

```diff
@@ -326,7 +326,7 @@ function pidofFTL()
             <!-- Sidebar user panel -->
             <div class="user-panel">
                 <div class="pull-left image">
-                    <img src="img/logo.svg" alt="Pi-hole logo" width="45" height="67" style="height: 67px;">
+                    <img src="img/logo.svg" alt="Pi-hole logo" width="45" height="67" style="height: 90px;">
```

Comment:
Why is the logo resized in height, is the aspect ratio still preserved?

When you want to do this, also edit the HTML height:
```suggestion
                    <img src="img/logo.svg" alt="Pi-hole logo" width="45" height="90" style="height: 90px;">
```

Reply:
@DL6ER 
The logo height is the only height affecting the parent (`.user-panel`) size, because the other <div> child has `position: absolute`.
I have a solution, but I'm waiting this PR to finish first.

Reply:
Now that I tried it myself, I see that the change was necessary as the temperature got its own line to make room for a possible long custom port text. It wasn't clear to me from the PR description or comments why this was needed. Now that I see it, I realize it was implicitly there in the screenshot.